data.json

```diff
@@ -1648,5 +1648,11 @@
     "githubUsername": "avneesh0612",
     "imageUrl": "https://user-images.githubusercontent.com/76690419/143516958-4e176034-cbce-45dd-8667-f3e9208e5c19.png",
     "issueNumber": 863
+  },
+  {
+    "name": "Anish De",
+    "githubUsername": "AnishDe12020",
+    "imageUrl": "https://user-images.githubusercontent.com/63192115/143732396-ecabab2a-e99c-402b-b568-3c562ae1d574.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
```

Reply:
Makes sense